src/clients/sidecar-openapi-specs/sidecar.openapi.yaml

```diff
@@ -737,6 +737,33 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Connection"
+    DataFormat:
+      description: |
+        The data format that represents the bytes of a Kafka message.
+
+        * AVRO: Apache Avro schema format.
+        * JSONSCHEMA: JSON schema format.
+        * PROTOBUF: Google Protocol Buffers schema format.
+        * JSON:
+            Bytes parsed as JSON.
+            The bytes did not contain a magic byte specifying a schema id, and the raw bytes
+            were successfully parsed into a JSON value.
+        * UTF8_STRING:
+            Bytes parsed as a UTF-8 string (meaning the bytes were not parsed as valid JSON).
+        * RAW_BYTES:
```

Comment:
Need to clarify that 'raw bytes' isn't actually sent, Base64 of the original kafka payload bytes are sent.